### PR TITLE
add gh issue templates

### PR DESCRIPTION
**Pull request by Ebedthan**
I have added Github Issue templates to ease the submission of suggestions or bug reports.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
```
Comment (natir) [inline .github/ISSUE_TEMPLATE/bug_report.md:16]: Not realy relevant in our case

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
```
Comment (natir) [inline .github/ISSUE_TEMPLATE/bug_report.md]: This example isn't really relevant in our case.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
```
Comment (natir) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: Same as above

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
```
Comment (natir) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Same as above, but maybe we can keep it

**Review by natir — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (Ebedthan) [reply .github/ISSUE_TEMPLATE/bug_report.md]: I've made some changes as you suggested. Hope it's more relevant now.

**Comment by natir** [comment]
Thanks for your work it's seems realy nice.

**Comment by natir** [comment]
I want solve MSRV (PR #48) before merge this but it's ok for me.

**Review by luizirber — APPROVED** [review]

**Pull request merged**